requirements-dev.txt

```diff
@@ -16,5 +16,4 @@ progressbar==2.5
 -e git+https://github.com/qld-gov-au/ckanext-report.git@develop#egg=ckanext-report
 -e git+https://github.com/qld-gov-au/ckanext-qa.git@develop#egg=ckanext-qa
 -e git+https://github.com/qld-gov-au/ckanext-ytp-comments.git@develop#egg=ckanext-ytp-comments
--e git+https://github.com/qld-gov-au/ckanext-datarequests.git@develop#egg=ckanext-datarequests
```

Comment:
so we can have this pass testing without datarequests? 
any luck on removing the other plugins?

Reply:
Actually, that entry was a duplicate of line 11.